hook/DestIPFoundHook.js

```diff
@@ -281,7 +285,9 @@ class DestIPFoundHook extends Hook {
     try {
       let intel;
       if (!skipReadLocalCache) {
-        intel = await intelTool.getIntel(ip);
+        const dIntel = await intelTool.getDomainIntel(domain);
```

Comment:
* intel:dns is just a cache of cloud result, should not replace the entire intel:ip.
* only skip querying cloud when there are already domain intel records for domain and sub domains.